test/integration/networks/networks_test.go

```diff
@@ -235,31 +217,25 @@ func TestNetworks(t *testing.T) {
 
 	cidrRanges := map[string]map[string][]string{
```

Comment:
if there is only `svpc` it could be just `map[string][]string` for all these items